app/policies/client_policy.rb

```diff
@@ -1,2 +1,42 @@
 class ClientPolicy < ApplicationPolicy
+
+  class Scope < Scope
+    def resolve
+      if user.has_role?('admin')
+        all
+      else
+        none
+      end
+    end
+  end
+
+  def index?
+    is_admin
```

Comment:
There is a `admin?` method on ApplicationPolicy that you could use here instead of this local method.